src/FilterAdder.js

```diff
@@ -13,24 +14,27 @@ let FilterAdder = ({
   node,
   path,
   fields,
-  theme: { AdderPicker },
   uniqueFields,
-  label = 'Add Custom Filter',
+  theme: { Icon },
+  Picker = ModalPicker,
```

Comment:
I'm not sure why this needs to be a prop 🤔

Reply:
The situation with FilterAdder and its pickers is so "it's complicated" that I'm definitely going to forget all the details as soon as I'm not touching this every day. I'll go ahead and drop a summary here for future reference:

- the theme key for a picker component is NestedPicker, which is used inside ModalPicker (and by default is a nested list of items with a select-like API)
- ModalPicker is not included as a theme key (or a part of the greyVest lib) because it consumes the NestedPicker (formerly just Picker) theme key for its internal picker
  - ModalPicker is composed of a picker like NestedPicker, a modal, and a button to summon the modal, all of which are themed
- because ModalPicker is not in the greyVest library or theme, FilterAdder can't get it from consuming the theme, so it imports it directly instead
- according to our stories, FilterAdder should be able to use a Select component instead of its normal ModalPicker, since those two should have the same API
  - the Picker prop was added to allow overriding ModalPicker for this specific reason